neutromeratio/ani.py

```diff
@@ -41,17 +42,33 @@ def add_restraint(self, restraint:Restraint):
 
         self.list_of_restraints.append(restraint)
 
-    def minimize(self, ani_input):
+    def minimize(self, ani_input, hybrid=False):
```

Comment:
Could we add a docstring clarifying what shape `ani_input['ase_mol']` is? Seems like this can be both one or multiple conformations...